ably-ios/ARTClientOptions.h

```diff
@@ -14,24 +14,32 @@ ART_ASSUME_NONNULL_BEGIN
 
 @interface ARTClientOptions : ARTAuthOptions
 
-@property (readonly, getter=getRestHost) NSString *restHost;
-@property (readonly, getter=getRealtimeHost) NSString *realtimeHost;
+@property (readwrite, strong, nonatomic, getter=getRestHost) NSString *restHost;
```

Comment:
Does it need one? I assumed this synthesizes a `_restHost` variable with a setter.